src/constants.js

```diff
@@ -26,3 +26,6 @@ export const ALL_CLASSES = [
   'MAGE',
   'WARLOCK',
 ];
+
+export const DEFAULT_SEASONS = ['s1', 's2', 's3', 's4'];
```

Comment:
Shouldn't we call it ALL_SEASONS?

Reply:
nah because it's the default seasons checked in the filter. Default is all seasons. But we could update the default to s4 only later one for example.